plugin/admin/src/pages/App/index.js

```diff
@@ -16,7 +15,6 @@ const App = () => {
     <div>
       <Switch>
         <Route path={`/plugins/${pluginId}`} component={HomePage} exact />
```

Comment:
Do we even need the whole Switch/Route thing at this point? I think just (re)exporting the Homepage component, maybe wrapped in a div, would work just as well?

Reply:
Your right, it's not necessary